openvpn/process.go

```diff
@@ -102,8 +108,9 @@ func (process *Process) waitForShutdown(cmd *exec.Cmd) {
 	select {
 	// Wait for shutdown
 	case <-process.cmdShutdownStarted:
-		// Kill the server
-		if err := cmd.Process.Kill(); err != nil {
+		//First - shutdown gracefully
+		//TOOD - add timer and send SIGKILL after timeout?
+		if err := cmd.Process.Signal(syscall.SIGTERM); err != nil {
 			return
```

Comment:
investigate we we can print / pass error somehow.

Reply:
Maybe at least print, because we are in shutdown phase already.